socorro/external/postgresql/postgresqlalchemymanager.py

```diff
@@ -43,91 +33,87 @@ def __init__(self, sa_url, logger, autocommit=False):
+        logging.debug('setting body checks off')
         self.session.execute('SET check_function_bodies = false')
 
-    def create_types(self):
-        types_dir = os.path.normpath(os.path.join(
+    def load_raw_sql(self, directory):
+        logging.debug('trying to load raw sql with dir %s' % directory)
+        sqlfile_path = os.path.normpath(os.path.join(
             __file__,
             '..',
-            'raw_sql/types',
+            'raw_sql',
+            directory,
             '*.sql'
         ))
-        for myfile in sorted(glob(types_dir)):
-            custom_type = open(myfile).read()
+        for myfile in sorted(glob(sqlfile_path)):
+            logging.debug('trying to load file %s' % myfile)
+            raw_sql = open(myfile).read()
             try:
-                self.session.execute(custom_type)
+                self.session.execute(raw_sql)
             except exc.SQLAlchemyError, e:
-                print "Something went horribly wrong: %s" % e
+                logging.error("Something went horribly wrong: %s" % e)
                 raise
         return True
 
     def create_tables(self):
+        logging.debug('creating all tables')
         status = self.metadata.create_all()
         return status
 
-    def create_procs(self):
-        procs_dir = os.path.normpath(os.path.join(
-            __file__,
-            '..',
-            'raw_sql/procs',
-            '*.sql'
-        ))
-        for file in sorted(glob(procs_dir)):
-            procedure = open(file).read()
-            try:
-                self.session.execute(procedure)
-            except exc.SQLAlchemyError, e:
-                print "Something went horribly wrong: %s" % e
-                raise
-        return True
-
-    def create_views(self):
-        views_dir = os.path.normpath(os.path.join(
-            __file__,
-            '..',
-            'raw_sql/views',
-            '*.sql'
-        ))
-        for file in sorted(glob(views_dir)):
-            procedure = open(file).read()
-            try:
-                self.session.execute(procedure)
-            except exc.SQLAlchemyError, e:
-                print "Something went horribly wrong: %s" % e
-                raise
-        return True
-
     def bulk_load(self, data, table, columns, sep):
+        logging.debug('bulk loading data')
         connection = self.engine.raw_connection()
         cursor = connection.cursor()
         cursor.copy_from(data, table, columns=columns, sep=sep)
         connection.commit()
 
-    def set_default_owner(self, database_name):
+    @ignore_if_on_heroku
+    def set_default_owner(self, database_name, username):
+        logging.debug('setting database %s owner to %s' % (
+            database_name, username
+        ))
         self.session.execute("""
-                ALTER DATABASE %s OWNER TO breakpad_rw
-            """ % database_name)
+                ALTER DATABASE %s OWNER TO %s
+            """ % (database_name, username))
```

Comment:
Shouldn't that be `self.session.execute("ALTER DATABASE %s ...", (database_name, username))`?
You know, in case someone tried to set their username to `'foo'; DROP TABLE some_table`